w3ctrace/state.go

```diff
@@ -109,3 +141,16 @@ func (st State) String() string {
 
 	return buf.String()
 }
+
+func filterEmptyItems(entries []string) []string {
+	var entriesWithoutEmptyItems []string
```

Comment:
As a suggestion, you could consider preallocating the slice here, since its max capacity is known.

Reply:
This part now has a different implementation, without extra memory allocation.